src/toil/fileStore.py

```diff
@@ -1603,7 +1607,8 @@ def open(self, job):
                                            disk=diskUsed,
                                            humanRequestedDisk=bytes2human(jobReqs),
                                            requestedDisk=jobReqs))
-            self.logToMaster(logString, level=logging.INFO)
+            self.logToMaster(logString, level=(logging.DEBUG if self.jobName.startswith(self.debugJobNames)
+                                               else logging.INFO))
             if diskUsed > jobReqs:
```

Comment:
I would possibly change this to be if/else since this message and the one above are redundant